server/portal/apps/projects/models/base.py

```diff
@@ -578,7 +582,7 @@ def __str__(self):
         return str(self.value)
 
 
-class ProjectSystemSerializer(BaseAgaveSystemSerializer):
+class ProjectSystemSerializer():
```

Comment:
I think we can also deprecate the `ProjectSystemSerializer`, since without the base class it will probably just throw an exception when it runs.